modules/azurerm/Azure-Container-Registry-Token/token.tf

```diff
@@ -0,0 +1,31 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+#
+# This software is the property of WSO2 LLC. and its suppliers, if any.
+# Dissemination of any information or reproduction of any material contained
+# herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+# You may not alter or remove any copyright or other notice from copies of this content.
+#
+# --------------------------------------------------------------------------------------
```

Comment:
Please update this to the Public License Header, similar to: https://github.com/wso2/azure-terraform-modules/blob/main/modules/azurerm/CDN-FrontDoor-Custom-Domain-Association/cdn_frontdoor_custom_domain_association.tf#L1